R/list-combine.R

```diff
@@ -65,15 +65,15 @@ list_rbind <- function(x, names_to = rlang::zap(), ptype = NULL) {
 check_list_of_data_frames <- function(x, error_call = caller_env()) {
   vec_check_list(x, call = error_call)
 
-  is_df <- map_lgl(x, is.data.frame)
+  is_df_or_null <- map_lgl(x, function(x) is.data.frame(x) || is.null(x))
 
-  if (all(is_df)) {
+  if (all(is_df_or_null)) {
     return()
   }
 
-  bad <- which(!is_df)
+  bad <- which(!is_df_or_null)
   cli::cli_abort(
-    "All elements of {.arg x} must be data frames. Elements {bad} are not.",
+    "Each element of {.arg x} must be either a data frame or NULL. Elements {bad} are not.",
```

Comment:
```suggestion
    "Each element of {.arg x} must be either a data frame or {.code NULL}. Elements {bad} are not.",
```

Maybe? Seems weird to not at least backtick null